v4: Add an actual `data-touch="false"` example in the carousel docs

patrickhlauke opened the pull request:
Backport of https://github.com/twbs/bootstrap/pull/32638

patrickhlauke committed:
v4: Add an actual `data-touch="false"` example in the carousel docs

Backport of https://github.com/twbs/bootstrap/pull/32638

patrickhlauke committed:
Remove data-ride from example carousel

patrickhlauke committed:
Correct attribute table explanation and remove slide class from example

i believe the correct option is `slide` here, not `ride`.
the `if "carousel"` bit was rather unclear/confusing

patrickhlauke committed:
Edits...

sorry, trying to make sense of our quirky carousel stuff, it seems some illogical things have been there ever since

patrickhlauke committed:
Add data-interval to suppress autoplay

patrickhlauke committed:
Tweak carousel options formatting

hashem213 commented:
xfcfv

mdo commented on site/content/docs/4.5/components/carousel.md:
```diff
@@ -208,6 +208,34 @@ Add `data-interval=""` to a `.carousel-item` to change the amount of time to del
 </div>
 {{< /example >}}
 
+### Suppressing touch swiping
```
```suggestion
### Disable touch swiping
```

mdo commented on site/content/docs/4.5/components/carousel.md:
```diff
@@ -208,6 +208,34 @@ Add `data-interval=""` to a `.carousel-item` to change the amount of time to del
 
+### Suppressing touch swiping
+
+By default, carousels support swiping left/right on touchscreen devices to move between slides. This can be explicitly suppressed using the `data-touch` attribute. The example below also does not include the `data-ride`, and has `data-interval="false"`, meaning that it won't autoplay.
```
```suggestion
Carousels support swiping left/right on touchscreen devices to move between slides. This can be disabled using the `data-touch` attribute. The example below also does not include the `data-ride` and has `data-interval="false"` so it doesn't autoplay.
```

patrickhlauke replied:
addressed with separate commit, as i also noticed that this could do with the extra word "attribute" after `data-ride`

patrickhlauke committed:
Update site/content/docs/4.5/components/carousel.md

Co-authored-by: Mark Otto <markd.otto@gmail.com>

patrickhlauke committed:
Tweak language

mostly based on @mdo's suggestion, but also adds the word "attribute" after `data-ride` for better flow/legibility

patrickhlauke commented:
once this is approved/merged, i'll revisit v5 (since the original https://github.com/twbs/bootstrap/pull/32638 has already been merged) and make matching changes there @mdo
